### PR TITLE
Shilpanikte hometown

### PR DESCRIPTION
**Pull request by shilpanikte**
This resolves issue #11

### CODE REVIEW

File: shilpanikte-Shrewsbury.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+##Shrewsbury MA
```
Comment (shilpanikte) [inline shilpanikte-Shrewsbury.md]: looks good

File: shilpanikte-Shrewsbury.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+### Great places to eat
```
Comment (kmpetersondell) [inline shilpanikte-Shrewsbury.md]: @shilpanikte Adding comments to your review.

File: shilpanikte-Shrewsbury.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+
+### Activities
```
Comment (kmpetersondell) [inline shilpanikte-Shrewsbury.md]: @shilpanikte Anything fun to do here?

**Comment by shilpanikte** [comment]
Committing local changes.

**Pull request merged**